store/store.go

```diff
@@ -548,7 +548,7 @@ func (s *Store) RemoveACI(key string) error {
 		dirUid := int(fi.Sys().(*syscall.Stat_t).Uid)
 
 		if uid != dirUid && uid != 0 {
-			return fmt.Errorf("permission denied, are you root or the owner of the image?")
+			return fmt.Errorf("permission denied, are you root or the owner of the image? User: uid=%d euid=%d. Image directory: uid=%d", uid, os.Geteuid(), dirUid)
```

Comment:
OK. But here we're actually checking the real uid to reject the deletion, not the effective uid. This should be fine since rkt is not a setuid program and we never call `seteuid`.